server/node/src/main/java/org/kaaproject/kaa/server/admin/client/util/Utils.java

```diff
@@ -174,14 +176,15 @@ public static String millisecondsToDateTimeString(long millis) {
    * @return boolean 'true' if email address is valid
    */
   public static boolean validateEmail(String mail) {
-    boolean result = false;
-    if (mail != null && mail.length() != 0) {
-      if (mail.indexOf('@') != INCORRECT_IDX
-          && mail.indexOf('.') != INCORRECT_IDX) {
-        result = true;
-      }
+    if (mail == null || mail.length() == 0) {
+      return false;
```

Comment:
looks good, 
but maybe in future we should return more complex error object, like

if email valid - return null
if email invalid return

{
  error: true,
  errorNumber: Integer,
  errorText: String
}

where errorText will have detailed explanation what was wrong in email, or something like

"Email could not be empty string"

this will help users better understand what we want from them